File: app/src/main/java/com/itachi1706/busarrivalsg/BusServices.java
package com.itachi1706.busarrivalsg;

/**
 * Created by Kenneth on 17/6/2015
 * for SingBuses in package com.itachi1706.busarrivalsg
 */
public class BusServices {

    private String serviceNo,operator,stopID;
    private BusStatus currentBus,nextBus;

    public BusStatus getNextBus() {
        return nextBus;
    }

    public void setNextBus(BusStatus nextBus) {
        this.nextBus = nextBus;
    }

    public BusStatus getCurrentBus() {
        return currentBus;
    }

    public void setCurrentBus(BusStatus currentBus) {
        this.currentBus = currentBus;
    }

    public String getOperator() {
        return operator;
    }

    public void setOperator(String operator) {
        this.operator = operator;
    }

    public String getServiceNo() {
        return serviceNo;
    }

    public void setServiceNo(String serviceNo) {
        this.serviceNo = serviceNo;
    }

    public String getStopID() {
        return stopID;
    }

    public void setStopID(String stopID) {
        this.stopID = stopID;
    }
}

File: app/src/main/java/com/itachi1706/busarrivalsg/objects/BusServices.kt
package com.itachi1706.busarrivalsg.objects

/**
 * Created by Kenneth on 17/6/2015
 * for SingBuses in package com.itachi1706.busarrivalsg
 */
data class BusServices(var serviceNo: String = "", var operator: String = "", var stopID: String = "", var currentBus: BusStatus? = null, var nextBus: BusStatus? = null,
    var isObtainedNextData: Boolean = false, var time: Long = 0, var stopName: String = "", var isSvcStatus: Boolean = false, var subsequentBus: BusStatus? = null)
